test/runs.test.js

```diff
@@ -377,3 +379,57 @@ describe('Run methods', () => {
         });
     });
 });
+
+describe('Redirect run logs', () => {
+    let baseUrl;
+
+    beforeAll(async () => {
+        // Ensure that the tests that use characters like á are correctly decoded in console.
+        process.stdout.setDefaultEncoding('utf8');
+        const server = await mockServer.start();
+        baseUrl = `http://localhost:${server.address().port}`;
+    });
+
+    afterAll(async () => {
+        await Promise.all([mockServer.close()]);
+    });
+
+    let client;
+    beforeEach(async () => {
+        client = new ApifyClient({
+            baseUrl,
+            maxRetries: 0,
+            ...DEFAULT_OPTIONS,
+        });
+    });
+    afterEach(async () => {
+        client = null;
+    });
+
+    const testCases = [
+        { fromStart: true, expected: MOCKED_ACTOR_LOGS_PROCESSED },
+        { fromStart: false, expected: MOCKED_ACTOR_LOGS_PROCESSED.slice(1) },
+    ];
+
+    describe('run.getStreamedLog', () => {
+        test.each(testCases)('getStreamedLog $fromStart', async ({ fromStart, expected }) => {
```

Comment:
```suggestion
        test.each(testCases)('getStreamedLog fromStart:$fromStart', async ({ fromStart, expected }) => {
```

<img width="414" height="131" alt="image" src="https://github.com/user-attachments/assets/bb4d7b47-7ba7-4811-8d45-c4bdc7c0a439" />